packages/storybook/stories/va-accordion.stories.jsx

```diff
@@ -73,6 +73,41 @@ const TemplateSubheader = args => {
   );
 };
 
+const I18nTemplate = args => {
+  const { headline, level, ...rest } = args;
+  const [lang, setLang] = useState('en');
+
+  useEffect(() => {
+    document.querySelector('main').setAttribute('lang', lang);
+  }, lang);
```

Comment:
Getting a local error that syntax should be `}, [lang]);`

Reply:
Can you show that screenshot? I'm not getting it so I'm having a hard time understanding what's wrong.

Reply:
<img width="415" alt="Screen Shot 2022-04-15 at 6 28 31 PM" src="https://user-images.githubusercontent.com/11822533/163649449-c052731a-dfab-47a4-bddf-97b1b5eae9ea.png">

I don't think the changing language buttons not working is related. This is more of a React useEffect warning that we should adjust https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect

Reply:
Ohhhh, I see - so we need `[lang]` instead of `lang` for that second arg. I'll make the update, good catch :+1: